src/content/development/_index.en.md

```diff
@@ -18,3 +18,14 @@ pre = "<b>4. </b>"
 * [Working with Shipyard](shipyard)
   * [Adding Shipyard to a Project](shipyard/first-time)
   * [Advanced Features](shipyard/advanced)
+
+### Research notice
+Please note that this repository is participating in a study into sustainability
+ of open source projects. Data will be gathered about this project's github repositories for
+ approximately the next 12 months, starting from September 2020.
```

Comment:
If this is supposed to be the date at which the notice is added to the docs, it should be

```suggestion
 approximately the next 12 months, starting from September 2021.
```